packages/near-membrane-base/src/membrane.ts

```diff
@@ -611,6 +644,44 @@ export function createMembraneMarshall() {
             return fn;
         }
 
+        function isLiveObject(value: any): boolean {
+            if (value === null || value === undefined) {
+                return false;
+            }
+            if (typeof value === 'object') {
+                if (
+                    // Arrays are marked as live in the BoundaryProxyHandler constructor.
+                    // isArrayOrNotOrThrowForRevoked(value) ||
+                    ArrayBufferIsView(value) ||
```

Comment:
This can be called with anything... objects, functions, etc... and it is pretty fast.... so maybe it should be alone in the check for quickness. Something like this at the top of the method:

```
if (ArrayBufferIsView(value)) { return true; }
```

Reply:
Done